apps/asap-server/test/controllers/users.test.ts

```diff
@@ -0,0 +1,532 @@
+        })
+        .reply(
+          200,
+          fixtures.buildUserGraphqlResponse({
+            teams: [
+              {
+                role: 'Lead PI (Core Leadership)',
+                approach: null,
+                responsibilities: null,
+                id: [
+                  {
+                    id: 'team-id-1',
+                    created: '2020-09-23T20:45:22Z',
+                    lastModified: '2020-10-26T15:33:18Z',
+                    data: null,
+                    flatData: {
+                      applicationNumber: 'applicationNumber',
+                      projectTitle: 'Awesome project',
+                      displayName: 'Jackson, M',
+                      proposal: [{ id: 'proposal-id-1' }],
+                      skills: [],
+                      outputs: [],
+                    },
+                  },
+                ],
+              },
+              {
+                role: 'Collaborating PI',
+                responsibilities: 'I do stuff',
+                approach: 'orthodox',
+                id: [
+                  {
+                    id: 'team-id-3',
+                    created: '2020-09-23T20:45:22Z',
+                    lastModified: '2020-10-26T15:33:18Z',
+                    data: null,
+                    flatData: {
+                      applicationNumber: 'applicationNumber',
+                      projectTitle: 'Another Awesome project',
+                      displayName: 'Tarantino, M',
+                      proposal: [{ id: 'proposal-id-2' }],
+                      skills: [],
+                      outputs: [],
+                    },
+                  },
+                ],
+              },
+            ],
+          }),
+        );
+
+      const result = await users.update(userId, {
+        teams: [
+          {
+            id: 'team-id-1',
+            responsibilities: '',
+            approach: '',
+          },
+          {
+            id: 'team-id-3',
+            responsibilities: 'I do stuff',
+            approach: 'orthodox',
+          },
+        ],
+      });
+
+      expect(result.teams).toEqual([
+        {
+          id: 'team-id-1',
+          displayName: 'Jackson, M',
+          proposal: 'proposal-id-1',
+          role: 'Lead PI (Core Leadership)',
+        },
+        {
+          displayName: 'Tarantino, M',
+          id: 'team-id-3',
+          proposal: 'proposal-id-2',
+          role: 'Collaborating PI',
+          responsibilities: 'I do stuff',
+          approach: 'orthodox',
+        },
+      ]);
+    });
+  });
+
+  describe('updateAvatar', () => {
+    test('Should throw when sync asset fails', async () => {
+      nock(config.baseUrl)
+        .post(`/api/apps/${config.appName}/assets`)
+        .reply(500);
+
+      await expect(
+        users.updateAvatar('user-id', Buffer.from('avatar'), 'image/jpeg'),
+      ).rejects.toThrow();
+    });
+
+    test('should return 500 when fails to update user', async () => {
```

Comment:
probably not a 500 as this is http specific and our controller should really just throw a more descriptive error. 

But since you're not really asserting the error message in these tests anyway i think just a generic 'error' should do